src/platform/x86/x86_64/start.S

```diff
@@ -0,0 +1,159 @@
+#include "chal/shared/cos_config.h"
```

Comment:
If this replaces loader.S, then remove loader.s. However, there is some code in there that we likely want (smp).

I don't see a reason to not just move this into loader.S and keep the file names consistent with 32 bit.